diff --git a/dist/components/Marquee.d.ts b/dist/components/Marquee.d.ts
index d0dad9c893d36a2df916d136e9458ed913c5514d..6700dce9920ecf79ed349ea3b463efdda505b46e 100644
--- a/dist/components/Marquee.d.ts
+++ b/dist/components/Marquee.d.ts
@@ -66,7 +66,7 @@ interface MarqueeProps {
      * Type: Array<number> of length 3
      * Default: [255, 255, 255]
      */
-    gradientColor?: [number, number, number];
+    gradientColor?: [number, number, number] | null;
     /**
      * The width of the gradient on either side
      * Type: string
diff --git a/dist/index.js b/dist/index.js
index 8aa3ef33e80cc750f6250ffdfea99feb8e613048..b949fde87daa74741d8d53d668a04a037db9154a 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -77,14 +77,14 @@ var Marquee = function (_a) {
         setIsMounted(true);
     }, []);
     // Gradient color in an unfinished rgba format
-    var rgbaGradientColor = "rgba(" + gradientColor[0] + ", " + gradientColor[1] + ", " + gradientColor[2];
+    var rgbaGradientColor = gradientColor ? "rgba(" + gradientColor[0] + ", " + gradientColor[1] + ", " + gradientColor[2] : '';
     // Animation duration
     var duration = marqueeWidth < containerWidth
         ? containerWidth / speed
         : marqueeWidth / speed;
     return (React__default['default'].createElement(React.Fragment, null, !isMounted ? null : (React__default['default'].createElement("div", { ref: containerRef, style: __assign(__assign({}, style), (_b = {}, _b["--pause-on-hover"] = !play || pauseOnHover ? "paused" : "running", _b["--pause-on-click"] = !play || (pauseOnHover && !pauseOnClick) || pauseOnClick ? "paused" : "running", _b)), className: className + " marquee-container" },
         gradient && (React__default['default'].createElement("div", { style: (_c = {},
-                _c["--gradient-color"] = rgbaGradientColor + ", 1), " + rgbaGradientColor + ", 0)",
+                _c["--gradient-color"] = gradientColor ? rgbaGradientColor + ", 1), " + rgbaGradientColor + ", 0)" : 'unset',
                 _c["--gradient-width"] = typeof gradientWidth === "number"
                     ? gradientWidth + "px"
                     : gradientWidth,